src/watcher.js

```diff
@@ -86,7 +86,7 @@ module.exports = class MonoplasmaWatcher extends EventEmitter {
      */
     async start(config) {
         await throwIfSetButNotContract(this.eth, config.contractAddress, "contractAddress from initial config")
-        this.log = debug("CPS::watcher::" + config.contractAddress)
+        this.log = log.extend(config.contractAddress)
```

Comment:
@jtakalai the `extend` API is pretty neat https://www.npmjs.com/package/debug#extend

Reply:
heh, ok. I guess it makes it more uniform